### ISSUE TITLE
Logits option for find_problem_type() function

### ISSUE THREAD

**Issue by orhunguley** [CBICA/GaNDLF#169]
**Is your feature request related to a problem? Please describe.**
It is related to classification tasks.

**Describe the solution you'd like**
find_problem_type() function should also search for "logits" option for classification tasks.
